model/src/w3wavemd.F90

```diff
@@ -1762,27 +1763,47 @@ SUBROUTINE W3WAVE ( IMOD, ODAT, TEND, STAMP, NO_OUT &
 #endif
 
 #ifdef W3_MEMCHECK
-       write(740+IAPROC,*) 'memcheck_____:', 'WW3_WAVE TIME LOOP 13'
+       write(40000+IAPROC,*) 'memcheck_____:', 'WW3_WAVE TIME LOOP 13'
        call getMallocInfo(mallinfos)
-       call printMallInfo(IAPROC,mallInfos)
+       call printMallInfo(IAPROC+40000,mallInfos)
 #endif
 !
-        IF ( FLSOU .and. LPDLIB) THEN
-!
-#ifdef W3_OMP0
-!$OMP PARALLEL DO PRIVATE (JSEA,ISEA,IX,IY) SCHEDULE (DYNAMIC,1)
+#ifdef W3_PDLIB
+        IF ( FLSOU .and. LPDLIB .and. FSSOURCE) THEN
 #endif
+
+!!#ifdef W3_OMP0
```

Comment:
@aliabdolali I thought this was going to be handled separately?

Reply:
yes, we will revert it back to the develop and will handle separately.

Reply:
done